src/v0/destinations/pardot/networkHandler.js

```diff
@@ -46,6 +46,20 @@ const getStatus = (code) => {
 const pardotRespHandler = (destResponse, stageMsg) => {
   const { status, response } = destResponse;
   const respAttributes = response['@attributes'];
+
+  // to handle errors like service unavilable, wrong url, no response
```

Comment:
Lets add inline documentation from pardot here

Reply:
in line 63, we are fetching stat from respAttributes, if it is not present will cause coding bug. Adding these scenarios to cover cases like DNS not resolved and similar